toc_config.py

```diff
@@ -197,8 +198,8 @@ def parse_chapter(docname, doc, parent):
         title_date_match = title_date_re.match(title)
         meta = first_meta(doc)
         status = u'hidden' if 'hidden' in meta else (
-            u'unlisted' if hidden else u'ready'
-        )
+                u'unlisted' if hidden else u'ready'
```

Comment:
You can use `git checkout -p upstream/master toc_config.py` and then say `y` to this part, to get the actual previous version. Of course, you need aalto-letech as an upstream repo (e.g. `git remote add upstream https://github.com/Aalto-LeTech/a-plus-rst-tools.git`).